src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Login from './pages/Login/Login'
+import {GlobalStyle} from "./theme/theme";
+
+
+const App: React.FC = () => {
+  return (
+      <GlobalStyle>
+        <Routes>
+          {/*<Route path="/" element={Login}/>*/}
```

Comment:
don't commit dummy/commented out code. remove it if it isn't needed anymore

Reply:
Just in case I have to create other routes